version

```diff
@@ -0,0 +1 @@
+0.2.0
```

Comment:
Do we need SDK version programmatically available? The application should have an version and pin dependencies, so not sure if this is useful.